### ISSUE TITLE
Support for the `text` property of the `photo` entity on the frontend

### ISSUE THREAD

**Issue by annando** [diaspora/diaspora#8297]
The protocol does support a `text` property for the `photo` entity: https://diaspora.github.io/diaspora_federation/entities/photo.html

To improve the accessibility it would be good if the frontend would add support for it. By now there doesn't seem to be a possibility to add a text when uploaded a photo. If not done, of course also the processing and displaying of this alternate text should be supported.

**Comment by brennanyoung** [comment]
image content without text alternative is a WCAG violation, so this is important.
Use of Diaspora in the public sector in many jurisdictions (North America, EU, Australia etc.) **requires** this feature.

Note: Decorative images do **not** need a text alternative, but should still be marked up properly.

The optimal implementation is:
GUI prompts for a text alternative in the upload dialog **OR** you can click a "decorative" checkbox which is unchecked by default.
If the image is implemented as `<img>` the text alternative is a value for the `alt` attribute. Purely decorative images should have a null alt attribute `alt=""`, which should be the consequence of checking the "decorative" checkbox.

Ideally should be possible to add alt text to an image you uploaded earlier, but this is a nice-to-have.
